src/Language/Haskell/LSP/TH/DataTypesJSON.hs

```diff
@@ -2848,16 +2848,16 @@ Registration Options: TextDocumentRegistrationOptions
 
 -}
 
-data MarkedString =
-  -- TODO: Add the plain string variant too
-  MarkedString
+data LanguageString =
```

Comment:
In the protocol,

    type MarkedString = string | { language: string; value: string };

I simply named { language: string; value: string } as LanguageString 

There is still a MarkedString in DatatypesJSON.

    type MarkedString = Either Text LanguageString